NEWS

```diff
@@ -128,7 +133,9 @@
 * Enhanced multi-thread test output
 #### Build
 * Fixed JUCX package publishing, so it will include support for ARM
-* Fixed ROCM building and testing
+* Fixed ROCm building and testing
+* Removed libnvidia-compute version dependency
+* Removed libibmad/libumad from default external libraries list
```

Comment:
Removed libibmad/libumad from default build configuration to avoid runtime dependency